cmake/conan.cmake

```diff
@@ -55,7 +55,7 @@ function(_get_msvc_ide_version result)
         set(${result} 15 PARENT_SCOPE)
     elseif(NOT MSVC_VERSION VERSION_LESS 1920 AND MSVC_VERSION VERSION_LESS 1930)
         set(${result} 16 PARENT_SCOPE)
-    elseif(NOT MSVC_VERSION VERSION_LESS 1930 AND MSVC_VERSION VERSION_LESS 1940)
+    elseif(NOT MSVC_VERSION VERSION_LESS 1930 AND MSVC_VERSION VERSION_LESS 1950)
```

Comment:
Just applies this change https://github.com/conan-io/cmake-conan/commit/6e5369d13720f22e07e31bb6b9018dbe60529fea

Reply:
Actually it is better to upgrade to Conan 2, but it would take much more time :)